articles/sentinel/azure-sentinel-billing.md

```diff
@@ -198,6 +199,12 @@ With ADX, you can store data at a lower price, but still explore the data using
 
 For more information, see [Integrate Azure Data Explorer for long-term log retention](store-logs-in-azure-data-explorer.md).
 
+#### Use Data Collection Rules for your Windows Security Events
```

Comment:
@nayef-yassin use title caps here:
#### Use data collection rules for your Windows security events